package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,7 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_SIXTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
Looks like the deprecation warning was something older. I thought you introduced it in this PR. Did we by accident reschedule some deprecations now? Looking over the diff yes we did reschedule some deprecations.

Could you go over that again @dotsdl ?

We need a little bit more formal way to deal with deprecations and how to check that we remove functions for the targeted releases.